tests/cmds/test_cases.py

```diff
@@ -273,9 +270,9 @@ def test_file_events_add_calls_add_event_with_expected_params(runner, cli_state)
 
```

Comment:
On line 229, it seems to me like the wrong method is being mocked ... I don't understand what is going. `get_all` does not get called in `show`, as far as I can tell.

shouldnt we be mocking `cases.get()` instead?

Reply:
the test method at line 229, `test_show_prints_expected_data_with_include_file_events_option` ,  refers to `file-events` `get_all`, so its fine. When `--include-file-events` is passed we want all the events listed against the case.